src/main/kotlin/models/Channel.kt

```diff
@@ -18,7 +18,7 @@ object Channel : Table() {
 
     // payload defined
     val creatorId = varchar("creatorId", 36)
-    val teamId = varchar("teamId", 36)
+    val teamId = (varchar("teamId", 36) references Team.id)
```

Comment:
I'm glad you were able to find a way to make a foreign key here. The docs for this didn't have the best description for how to do it.